media/options-widget.css

```diff
@@ -57,6 +65,11 @@
   border-top: 1px solid var(--vscode-widget-border);
 }
 
+.core-options input:disabled,
```

Comment:
Could you please move the `input:disabled` changes to https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/blob/main/media/theme/components/input.css so that we can use it everywhere?

and also use the css `cursor: not-allowed;` please for the [input](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/blob/main/media/theme/components/input.css) and [button](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/blob/main/media/theme/components/button.css) when they are disabled.

Maybe you will also need to update: https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/blob/main/media/theme/extensions.css  

```css
.pm-top-label .p-inputtext-label {
  margin-bottom: 2px;
}

.pm-top-label:not(:has(*:disabled)) .p-inputtext-label {
  cursor: pointer;
}

.pm-top-label:has(*:disabled) {
  cursor: not-allowed;
}
```

Reply:
An alternative to the cursor changes would be simply not to render the 'load more' headers / footers when those options aren't available. But one of the two changes should be made - at the moment, we still show the pointer cursor over the headers, even though they can't be interacted with.